aeron-driver/src/main/c/aeron_retransmit_handler.c

```diff
@@ -170,15 +176,17 @@ int aeron_retransmit_handler_process_timeouts(
     return result;
 }
 
-aeron_retransmit_action_t *aeron_retransmit_handler_scan_for_available_retransmit(
+int aeron_retransmit_handler_scan_for_available_retransmit(
+    aeron_retransmit_action_t **actionp,
```

Comment:
I would make this the last parameter of the function call rather than the first.